crates/katana/rpc/rpc-types/src/trace.rs

```diff
@@ -72,3 +74,12 @@ impl From<CallInfo> for FunctionInvocation {
         })
     }
 }
+
+/// The type returned by the `saya_getTransactionExecutionsByBlock` RPC method.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct TxExecutionInfo {
+    /// The transaction hash.
+    pub hash: TxHash,
+    /// The transaction execution trace.
+    pub trace: TxExecInfo,
+}
```

Comment:
decided to attach the tx hash so that it can be easily consumed by `saya` here:

https://github.com/dojoengine/dojo/blob/855da3112c87faea87646db5a406ac77b4daf149/crates/saya/provider/src/rpc/mod.rs#L160-L177